docs/operating-scylla/admin-tools/task-manager.rst

```diff
@@ -45,7 +45,10 @@ task_stats
 - *sequence_number* - an operation number (per module). It is shared by all tasks in a tree. Irrelevant for cluster tasks;
 - *keyspace* - optional, name of a keyspace on which the task operates;
 - *table* - optional, name of a table on which the task operates;
-- *entity* - optional, additional info specific to the task.
+- *entity* - optional, additional info specific to the task;
+- *shard* - optional, shard id on which the task operates;
+- *start_time* - relevant only if state == created;
```

Comment:
I know this is pre-existing but, `start_time` is relevant for all other states as well, once a task was in `created` at least once`, no?